README.md

```diff
@@ -399,6 +399,10 @@ client.process_template template
 
 ## Upgrading
 
+#### past version 2.0
```

Comment:
@simon3z please note, we should make sure we bump to this version at some point (probably best with non backward compatible changes https://github.com/abonas/kubeclient/issues/199)